include/spdlog/sinks/database_logger_sink.h

```diff
@@ -0,0 +1,156 @@
+#pragma once
+#include <spdlog\sinks\sink.h>
+#include <spdlog/details/log_msg.h>
+#include <spdlog/common.h>
+#include <vector>
+#include <sqlite3.h>
+
+struct Column
+{
+	enum TableColumn
+	{
+		TimeStamp,
+		Level,
+		Message,
+		LoggerName,
+		ThreadId
+	};
+
+	Column(std::string columnName, TableColumn columnMap)
+	{
+		ColumnName = columnName;
+		ColumnMap = columnMap;
+	}
+
+	std::string ColumnName;
+	TableColumn ColumnMap;
+	std::string Value;
+};
+
+struct database_schema
+{
+	std::string TableName;
```

Comment:
To be consistent with spdlog's naming convention, please use snake_case for all variable names ("table_name", "column_map", "column_name", "value", "thread_id" ,etc..)